spinalcordtoolbox/centerline/core.py

```diff
@@ -13,7 +13,7 @@
 
 class ParamCenterline:
     """Default parameters for centerline fitting"""
-    def __init__(self, algo_fitting='bspline', degree=5, smooth=20, contrast=None, minmax=True):
+    def __init__(self, algo_fitting='bspline', degree=5, smooth=20, soft=0, contrast=None, minmax=True):
```

Comment:
Very small nitpick, but I think it's generally best practice to add new parameters to the very end of the function? 

That way, we can trust that existing calls that specify arguments positionally (without keywords): 

```python
ParamCenterline(algo_fitting, degree, smooth, contrast, mixmax)
```

Won't accidentally assign the wrong value to the wrong parameter. (In this case, `contrast` would get applied to `soft`.)

Reply:
Good point! Fixed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4026/commits/97dc82f07cc874850b3816c85f1ef1a5f2a62c7e